packages/web-extension/e2e/extension/config.ts

```diff
@@ -0,0 +1,135 @@
+// Review: this is copied from packages/wallet
```

Comment:
let's capture it as a separate task. It's fine for now given it's in the test scope

Reply:
captured in PR description